groundX-doc-pipeline/evaluation_geval.py

```
@@ -0,0 +1,295 @@
+            model="gpt-4o",
+            messages=[{"role": "user", "content": prompt}],
+            max_tokens=300,
+            temperature=0.3,
+        )
+        return resp.choices[0].message.content
+
+    def _gpt_direct(self, q: str, raw: bytes, mime: str) -> str:
+        img_uri = None
+        if mime.startswith("image/"):
+            img_uri = f"data:{mime};base64,{base64.b64encode(raw).decode()}"
+            print(f"DEBUG: Using image file directly")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)